api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -750,22 +750,45 @@ message ScopedRds {
       [(validate.rules).message = {required: true}];
 }
 
+// [#next-free-field: 6]
 message HttpFilter {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.network.http_connection_manager.v2.HttpFilter";
 
+  // [#not-implemented-hide:] Config source specifier for the late-bound filter configuration.
+  // The connection manager listener is placed into the warming state until all the initial filter configurations
+  // are received. If the initial filter configuration is not received due to a timeout or another xDS failure,
+  // the optional fallback configuration is used.
```

Comment:
What is the behavior in the case of failure but no fallback config is specified? I assume we would respond immediately with a 500 or similar?